bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/obs/ObsTargetCalculatorService.scala

```diff
@@ -51,7 +51,7 @@ object ObsTargetCalculatorService {
       // Ideally, if you hover over duration box in GUI, should say acquisition + science time OR not.
 
       // Since we need start < end explicitly, if the duration is None, we cannot use it.
-      val duration = b.duration getOrElse plannedTime.totalTime
+      val duration = b.duration getOrElse plannedTime.totalTime max 1 // REL-2786
```

Comment:
It would blow up in the `TargetCalculator`?  I'm not that familiar with it but it looks like a provision was made for a "single value" target calculator.  I wonder if the duration is 0 if the `TargetCalculator` should be `TargetCalculator(s, st, b.start)` at line 60.